src/adera/Machines/Rocket.cpp

```diff
@@ -34,6 +34,8 @@ using namespace adera::active::machines;
 using namespace osp::active;
 using namespace osp;
 
+const std::string SysMachineRocket::smc_name = "Rocket";
```

Comment:
With C++17, you don't need to define the variable in the cpp file.

You can use "static inline const std::string SysMachineRocket::smc_name = "Rocket";" in the class body itself.